README.md

```diff
@@ -1 +1,127 @@
-# Space-Travelers-Hub
\ No newline at end of file
+# Space Travelers' Hub
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+A corporation that offers services for commercial and scientific space flight has a web application called Space-Travelers-Hub. Users of the app will be able to reserve rockets and sign up for particular space missions.
+
+- Here is the preview
+<p>
+ <img src="./preview.gif" />
+</p>
+
```

Comment:
[Optional] Your readme show a proper documentation of your project. However, I recommend that you use the new Readme template that has been given to be used from now on at Microverse. Kindly follow this [link to implement](https://github.com/microverseinc/readme-template)